docs/components/store.md

```diff
@@ -250,6 +250,14 @@ Flags:
                                  The maximum series allowed for a single Series
                                  request. The Series call fails if this limit is
                                  exceeded. 0 means no limit.
+      --store.posting-group-max-key-series-ratio=100
```

Comment:
Maybe you could also add that one should look at `thanos_bucket_store_lazy_expanded_posting_groups_total` if they want to understand whether this value needs to be changed?

Reply:
Addressed in latest commit!